### PR TITLE
Updating test

### PR DESCRIPTION
**Pull request by mirtinika**
- Moving instructions from pdf to Markdown
- Updating package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,13 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
```
Comment (andreasabilio) [inline package.json:8]: Is this something we would like to have in our `@clio/testing`?

**Review by andreasabilio — APPROVED** [review]
LGTM 👍 but maybe you want somebody more involved in the process to approve as well 😄

File: instructions/instruction.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Exercise steps
+1. Notice that under app/public you can find a file called get.json. Fetch and use the data on that json file to build your implementation.
+
+2. Build a small menu with two entries: “Home” and “Bookmarks”. These two entries should give access to two different pages and the user should be able to switch between the two by clicking on the menu. (ref 1 in the design)
+
+3. Under the “Home” page, build a component that displays a list of cards based on the data you have retrieved in point one. (ref 2)
+
+4. Each card should be composed of (ref 3) an image, a title, a description, a sub list of topics and a bookmark button. You do not have to make it look pretty as long as it is functional.
+
+5. The user should be able to add and remove the movies to his bookmarks. To do so, he can click on the bookmark button (ref 4). The button should change state based on the movie being bookmarked or not. Normally you would save the state to the server but for this test you can keep the state locally as long as the application works.
```
Comment (Odder) [inline instructions/instruction.md]:
```suggestion
5. The user should be able to add and remove the movies to his bookmarks. To do so, he can click on the bookmark button (ref 4). The button should change state based on the movie being bookmarked or not. Normally you would save the state to the server but for this exercise you can keep the state locally as long as the application works.
```

File: instructions/instruction.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+![Clio logo](logo.png)
+
+Welcome to the Clio React test. The following test contains a small design that you can use for reference and multiple steps that you will illustrate what to do. You can work on each step in the order that you prefer.
```
Comment (Odder) [inline instructions/instruction.md]:
```suggestion
Welcome to the Clio React test. The following exercise contains a small design that you can use for reference and a few points that explains the requirements of the solution. You can work on each step in the order that you prefer.
```

**Review by Odder — COMMENTED** [review]
Great decision to move from the PDF to a Markdown file! ☀️

File: instructions/instruction.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+The test can be found at https://github.com/clio-me/react-test you can either download the folder or fork the project :)
```
Comment (Odder) [inline instructions/instruction.md]: Given this page is linked within the README.md file, does it make sense to have this back-link?

Comment (mirtinika) [reply instructions/instruction.md]: Thanks for feedback, I've removed the sentence

**Pull request merged**